test/AndroidManifest.xml

```diff
@@ -11,7 +11,6 @@
         android:targetPackage="org.sqldroid" />
 
     <application
-        android:icon="@drawable/ic_launcher"
```

Comment:
If we don't remove it, some auto compile/test is failing during GitHub checks because the ic_launcher.png file is not present in the repository. If we keep it, we need to add a binary icon file (ic_launcher.png) in the repository. What do you suggest?